index.html

```diff
@@ -23,60 +23,85 @@
      </div>
   </header>  
   <main id="section-main">
-        <h1>AMBROSE <br> ISIGBA</h1>
-        <p>Hello! I am a passionate software developer! my goal
+     <div id ="section-main-text">
+         <h1>AMBROSE ISIGBA</h1>
+         <p>Hello! I am a passionate software developer! my goal
            is to help you build a quality product(s), features 
            and/or website with international standard. You are 
-           highly invited to take a tour of my works.If you fall in 
-            love with  my work gallery, and you are planning for one or 
-           you need a code mentor. AMBROSE's CODELAB get you covered</p>
-        <button type="button">Start colaboration</button>
-        <div id="second-section-bg-img"></div>
-  </main>
-  <section id="project-sections">
-     <h2>Projects</h2>
+           highly invited to take a tour of my works.</p>
+          <button type="button">Start colaboration <img id="btn-imgA" src="./images/image_dots.png" alt=""></button>
+      
+        </div>
+        <div id="section-main-img2"> 
+      <div id="second-section-bg-img"><img src="./images/Shape1b.png" alt=""></div>
+      <div><img  src="./images/image-geometry1.png" alt=""></div>
+    </div>
+    </main>
+   <section id="project-sections">
      <div id="project-container">
+      <h2>Projects</h2>
        <div class="full-section-div">
-         <div id="img-div1" class="imj-project-section"><img id="full-section-div1" src="./images/desktop7.png" alt=""></div>
+         <div id="img-div1" class="imj-project-section">
+          <img id="desktop-img1a" src="./images/image_dots.png" alt="">
+           <img id="full-section-div1" src="./images/desktop7.png" alt="">
+        </div>
          <div id="list-of-project1" class="list-of-project">
           <h4>Project names goes here</h4>
           <ul>
                <li class="project-list-1">HTML/CSS</li>
                <li class="project-list-2">Ruby on Rails</li>
                <li class="project-list-3">Javascricpt</li>
+               <li class="project-list-4">+1</li>
+
            </ul>
            <div class="project-link"><a href="#">See this project</a>
```

Comment:
- [ ] As the previous reviewer mentioned your "See this project" button should be a button and behave like one(with hover, active styles, etc). and wrapping it into a <div> is not the best practice to use a button. Kindly fix it and try to match it with the style that the Figma template provides.

Reply:
Done